nextjs.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  root: true,
+  extends: [
+    'next/core-web-vitals',
+    './react', // Keep next-to-last: ensures our TS overrides have priority
```

Comment:
I think this should be `Keep last` instead of `next-to-last`. Sorry I know you just copy-pasted my mistake 😅 This is from before I moved the prettier extension in the typescript config.